testkit/testkit.go

```diff
@@ -57,7 +57,7 @@ type TestKit struct {
 
 // NewTestKit returns a new *TestKit.
 func NewTestKit(t testing.TB, store kv.Storage) *TestKit {
-	require.True(t, intest.InTest, "you should add --tags=intest when to test")
+	require.True(t, intest.InTest, "you should add --tags=intest(in 'Go tool arguments' if using GoLand) when to test")
```

Comment:
Some developers are using vs code. I think You can link the [document](https://pingcap.github.io/tidb-dev-guide/get-started/setup-an-ide.html) here.